index.html

```diff
@@ -9,3 +13,12 @@
 	<script src="main.js">
 	</script>
 </head>
+<body>
+<div class="allBoxes">
+  <div class="answer" data-action="calculate">
+
+<div id="calculator">
+  
+</div>
+
+  </body>
```

Comment:
I see you jump-started this, try to get some buttons down in the body of your HTML.

Reply:
You got this! just a step at a time!